orchagent/neighorch.cpp

```diff
@@ -222,17 +243,16 @@ bool NeighOrch::addNextHop(const IpAddress &ipAddress, const string &alias)
     sai_status_t status = sai_next_hop_api->create_next_hop(&next_hop_id, gSwitchId, (uint32_t)next_hop_attrs.size(), next_hop_attrs.data());
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to create next hop %s on %s, rv:%d",
-                       ipAddress.to_string().c_str(), alias.c_str(), status);
+        SWSS_LOG_ERROR("Failed to create next hop %s, rv:%d",
```

Comment:
Please revert back to original log message

Reply:
It is not appropriate to revert this log to previous, since it will not contain full NH description.

Reply:
Previous log is having the relevant info. We don't have to repeat the same all over. Also, such logs may be used in existing deployments in the current pattern. Please revert back.

Reply:
@prsunny The previous log message only included IP address and Interface name of NH.  New format includes both IP address and Interface name of NH as well as MPLS label stack information.  Without the new format, the log output does not provide enough info to distinguish between NH with MPLS label 100 and MPLS label 200 on IP address 10.10.1.1 on Ethernet4.

Reply:
As I mentioned, you dont have to repeat it all over. You can just have the full info in one log and keep all the existing one as it is. We don't want to change these log message format for non-mpls scenario.

Reply:
For non-mpls scenario, the format changes from "x.x.x.x on EthernetX" to "x.x.x.x@EthernetX".  I really don't understand your objection to this format change, since it allows for better logging in general.  I will have to update all these same log messages anyway even if I preserve the text format, since ipAddress and alias are no longer parameters passed to the calling function.

Reply:
@prsunny New NextHopKey::toLogString() function maintains the original output.

Reply:
@prsunny All logs are reverted to previous format.